src/pages/console/data/content-management.mdx

```diff
@@ -68,12 +68,12 @@ Alternatively, you can save save *All About Dogs* before linking to an Author, a
 
 ## Limitations
 
-### Conflict Resolution
-Amplify Studio's data manager requires your app to have Conflict Resolution enabled. 
+### Conflict Resolution and DataStore
+Studio's Data Manager supports Amplify GraphQL APIs with and without DataStore. However, GraphQL APIs without DataStore have several minor limitations:
+- Data cannot be sorted by column
+- The search bar is disabled
```

Comment:
Is search disabled? Didn't we have search functioning in the autocompletes?

Reply:
The search bar is fully hidden for Data Manager when Conflict Resolution is disabled. I'm not sure if search _could_ work (we can explore that after launch), but even if it can, the search bar isn't there, so it can't be used.